README.md

```diff
@@ -93,6 +100,8 @@ platforms to receive funding from Open Source Pledge Members:
 - [Open Collective](https://opencollective.com/)
 - [Patreon](https://www.patreon.com/)
 - [Polar](https://www.polar.sh/)
+- [Software Freedom Conservancy (SFC)](https://sfconservancy.org/projects/current/)
```

Comment:
I wonder if, for the three platforms added, there might be better pages to link to. For example, for SFC, would it not be better to link to this page? https://sfconservancy.org/projects/apply/

Reply:
Changed; I think there should be a common pattern here - e.g. link to how to apply, or a list of projects. The fiscal host homepages are often not useful to figure these things out.

Reply:
Yep. I've also updated the SPI link to be more helpful. I personally find the Apache Incubator website very confusing and their requirements very involved, but I've left it in the list as-is.

Reply:
As a later PR; probably reach out to each of these and ask what landing page for potential new projects they want. It might be in their interest to include referrer parameters to show the value of OSSPledge